.github/workflows/ci.yml

```diff
@@ -26,4 +26,6 @@ jobs:
     - name: Install Dependencies
       run: npm install
     - name: Run
-      run: npm run ${{ matrix.version }}
+      run: |
+        COMMAND="${{ matrix.run }}"
+        npm run ${COMMAND:-__notfound__}
```

Comment:
Fail step if `${{ matrix.run }}` is undefined, because the command `npm run __notfound__` does not exist.